lib/ex_admin/form.ex

```diff
@@ -887,6 +886,31 @@ defmodule ExAdmin.Form do
     end
   end
 
+  defp build_checkboxes(conn, name, collection, opts, resource, model_name, errors, name_ids) do
+    theme_module(conn, Form).wrap_collection_check_boxes fn ->
+      for opt <- collection do
+        opt_id = Schema.get_id(opt)
+        name_str = "#{model_name}[#{name_ids}][#{opt_id}]"
+        cond do
```

Comment:
Selected was actually coming through here. it is used in the last line of the function.